src/webnn_native/ops/Resample2d.cpp

```diff
@@ -35,27 +35,32 @@ namespace webnn_native { namespace op {
         }
         mOptions.sizes = mSizes.data();
         mOptions.sizesCount = mSizes.size();
+
+        if (options != nullptr && options->axes != nullptr) {
+            mAxes.assign(options->axes, options->axes + options->axesCount);
+        }
+        mOptions.axes = mAxes.data();
+        mOptions.axesCount = mAxes.size();
```

Comment:
We can get the value from member variable, so those value don't set it back to option, please remove L.30-L.31,  L.36 - 37, L.42-43
```
std::vector<float> mScales;
        std::vector<int32_t> mSizes;
        std::vector<int32_t> mAxes;
```